training-manager.lic

```diff
@@ -122,19 +123,27 @@ class TrainingManager
     return if favor_count >= @settings.favor_goal
 
     if /could not/ =~ bput("tap my #{@settings.favor_god} orb", 'The orb is delicate', 'I could not find')
-      wait_for_script_to_complete('favor')
+      run_favors
       fput('stow my orb')
     elsif rub_orb?
       walk_to(5865)
       fput("get my #{@settings.favor_god} orb")
       fput('put my orb on altar')
       if favor_count + 1 < @settings.favor_goal
-        wait_for_script_to_complete('favor')
+        run_favors
         fput('stow my orb')
       end
     end
   end
 
+  def run_favors
```

Comment:
❓ Why is the use of `use_favor_altars` part of `training-manager` and not `favor`?

Reply:
because its making training-manager pass the arg to automatically use favor altars over using the puzzle, people using ;favor can call it with an arg to turn it on manually

Reply:
ok, so, use case here is, I want to use ;favor immortalname called from another script in the future to make truffenyi-commune-quest handle getting the 3 orbs at the start of the quest, that was the whole goal of adding altar favors. i'd rather it not be baked in to favor